assets/js/leaves.js

```diff
@@ -50,6 +50,11 @@ class Leaf {
     Array.from(selected).forEach((item) => {
       item.classList.remove(Leaf.selectedClass);
     });
+
+    // change url location hash to indicate no leaf is selected
+    // needs to be non-empty to avoid page reload
+    window.location.replace("#");
+    history.replaceState(null, "", window.hash);
```

Comment:
I really like when websites do this. Every object/view ideally should have a stable URL. Makes citation a lot more powerful. So: bravo!